sync.js

```diff
@@ -62,6 +62,8 @@ GlobSync.prototype._finish = function () {
         } catch (er) {
           if (er.syscall === 'stat')
             set[self._makeAbs(p)] = true
+          else if (er.code === 'ELOOP')
+            if (real) set[real] = true
           else
```

Comment:
added braces to fix the error. I'm going to have to dig back into the code to figure out exactly why we went with setting real... although based on your response at the end of this PR it may not be worth putting in the time.